### ISSUE TITLE
Embedded client and protobuf-net

### ISSUE THREAD

**Issue by smalldave** [EventStore/EventStore#410]
Guessing this is the same as #26

Tried to start a cluster using embedded vnodes. 
I didn't get any errors but no sign of replication and some 'strange' behaviour. 
I configured nlog (and finally remembered to click 'always copy' for the nlog.config file) which showed that something was expecting protobuf-net v2. 

Once I added that everything seems to work fine.

**Comment by ouro-ouroboros** [comment]
Ill fix this for a client release which is coming up today or tomorrow

Sent from my iPhone

On 6 Mar 2015, at 18:09, David Smith <notifications@github.com<mailto:notifications@github.com>> wrote:


Guessing this is the same as #26<https://github.com/EventStore/EventStore/issues/26>

Tried to start a cluster using embedded vnodes.
I didn't get any errors but no sign of replication and some 'strange' behaviour.
I configured nlog (and finally remembered to click 'always copy' for the nlog.config file) which showed that something was expecting protobuf-net v2.

Once I added that everything seems to work fine.

-
